src/QMCWaveFunctions/DiffWaveFunctionComponent.h

```diff
@@ -146,10 +146,10 @@ struct AnalyticDiffOrbital : public DiffWaveFunctionComponent
 
   void evaluateDerivatives(ParticleSet& P,
                            const opt_variables_type& optvars,
-                           std::vector<RealType>& dlogpsi,
-                           std::vector<RealType>& dhpsioverpsi);
-  void checkOutVariables(const opt_variables_type& optvars);
-  void resetParameters(const opt_variables_type& optvars);
+                           std::vector<ValueType>& dlogpsi,
+                           std::vector<ValueType>& dhpsioverpsi) override;
+  void checkOutVariables(const opt_variables_type& optvars) override;
+  void resetParameters(const opt_variables_type& optvars) override;
```

Comment:
I had to change `RealType` to `ValueType` in for `override` to work in both real and complex builds

Reply:
Nice catch